src/main/java/com/revo/deployr/client/util/RDataUtil.java

```diff
@@ -242,6 +252,8 @@ private static RData buildVectorFromJSON(String name, Map json) {
                             ex.printStackTrace();
                             dates.add(null);
                         }
+                    }else{
```

Comment:
Just move dates.add(null) outside and after the if-else, instead of repeating it in both